indico/modules/events/timetable/templates/balloons/_balloon_utils.html

```diff
@@ -33,3 +33,18 @@
         {% endif %}
     </dl>
 {% endmacro %}
+
+{% macro render_palette_picker(obj, endpoint, color_list) %}
+    <dl class="i-data-list thirty-seventy">
+        <dt>{% trans %}Color{% endtrans %}</dt>
+        <dd>
+            <a class="i-button icon-palette palette-picker palette-picker-trigger"
+               style="{{ obj.colors.css }}"
+               data-method="PATCH"
+               data-href="{{ url_for(endpoint, obj) }}"
+               data-palette={{ color_list | tojson }}
+               data-initial-color={{ obj.colors.css | tojson}}
```

Comment:
I found out that by using `| tojson | forceescape` I get a properly escaped output.